### DISCUSSION TITLE
FR: Distinguish update entities for custom integration and add-on.

### DISCUSSION THREAD

**Discussion by saya6k** [music-assistant/support#2608] [Feature requests and ideas]
<img width="1440" alt="스크린샷 2024-07-11 17 45 44" src="https://github.com/music-assistant/hass-music-assistant/assets/63517312/4499feae-991d-4dba-8323-e1ba288055cb">

For now, custom integration and add-on share the same name, so updating entities ends up with the same entity_id, with a suffix '_2' randomly defined by which installation was done first. I think they should have different entity_ids to distinguish them from each other, such as using the suffix 'addon', 'integration', or something similar by default.

**Comment by OzGav** [comment]
We will probably make some tweaks as this is easy to do. Also the plan is put the MA integration into core so you won’t see these updates

**Reply by saya6k** [reply]
Great! Looking forward to see Music Assistant official integration. I'll close the FR, then?
